wordpress-persistent-disks/mysql-service.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: mysql
+  labels:
+    app: mysql
+spec:
+  type: ClusterIP
```

Comment:
yeah I wanted to make it explicit, as the other wordpress-service.yaml has type:LoadBalancer.